src/test/.gitignore

```diff
@@ -0,0 +1 @@
+submit_command.sh
```

Comment:
Not sure as to why this is here...?

Reply:
the HammerSubmitCommand will always drop a self-contained command script and a log file in the run-directory. In haste, i just hard-coded the command script to submit_command.sh. when i run tests, that file got dropped in the hammer work-tree. 

if we ever want to run multiple hammerSubmitCOmmands in the same working-directory, this will have to be fixed (i'm assuming it will)